content/how-to/rackspace-fabric/Configure MFA/index.md

```diff
@@ -0,0 +1,73 @@
+---
+permalink: "Multi-Factor Authentication"
+audit_date: '2022-03-23'
+type: article
+title: "Multi-Factor Authentication"
+created_date: '2022-03-23'
+created_by: Rohit Rajput
+last_modified_date: '2022-03-23'
```

Comment:
Update the date to the date you address the rest of the feedback and push your changes